lib/index.js

```diff
@@ -171,10 +171,13 @@ bedrock.events.on('bedrock.start', async () => {
   // ready check
   const cancel = await bedrock.events.emit('bedrock-express.ready', server);
   // check if default attach canceled
-  if(cancel !== false) {
+  if(!bedrock.config.express.httpOnly || cancel !== false) {
     // attach express to TLS
     bedrockServer.servers.https.on('request', server);
   }
+  if(bedrock.config.express.httpOnly) {
+    bedrockServer.servers.http.on('request', server);
+  }
```

Comment:
I think that if the event was canceled (see above) we should not be attaching here. So, now reading this bit ... I think in total we want something like this:

```js
// ready check
const canceled = await bedrock.events.emit('bedrock-express.ready', server) === false;
// do not attach if ready was canceled
if(canceled) {
  return;
}
if(bedrock.config.express.httpOnly) {
  // attach express to HTTP only
  bedrockServer.servers.http.on('request', server);
} else {
  // attach express to TLS
  bedrockServer.servers.https.on('request', server);
}
```

Reply:
Done.